client/src/components/Dashboard/MyApps/MyAppsHeader.js

```diff
@@ -1,16 +1,32 @@
-import React from 'react';
+import React, { useState } from 'react';
 import { Row, Col, Button } from 'reactstrap';
-import { Heading } from '../../common';
+import { Heading, CustomModal, Select } from '../../common';
 
 const MyAppsHeader = () => {
+  const [modal, setModal] = useState(false);
+  const toggle = () => setModal(!modal);
+
   return (
     <Row>
       <Col sm={12} md={4} lg={4}>
         <Heading title="Connected Apps" />
       </Col>
       <Col sm={12} md={4} lg={4} />
       <Col sm={12} md={4} lg={4} className="mt-5 mb-5 align-center">
-        <Button outline color="primary">Connect New App</Button>
+        <Button outline color="primary" onClick={toggle}>Connect New App</Button>
+        <CustomModal
+          _toggle={toggle}
+          _modal={modal}
+          _button1text="Cancel"
+          _button2text="Connect"
+          _button2color="primary"
+          _modalTitle="Connect New App"
+          _modalBody={(
+            <>
```

Comment:
You don't have to use the angular brackets if there is only single chile.

Reply:
Noted.